src/features/user/Login.jsx

```diff
@@ -0,0 +1,97 @@
+import { signInWithEmailAndPassword } from "firebase/auth";
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import { Link, useNavigate } from "react-router-dom";
+import { auth } from "../../firebase/firebase";
+import { login } from "./userSlice";
+
+export default function Login() {
+  const navigate = useNavigate();
+  const [loginDetails, setLoginDetails] = useState({
+    email: "",
+    password: "",
+  });
```

Comment:
suggestion: you could use this, using useRef as well, it will prevent re render on every input change

Reply:
Will check this out, thank you!